.travis.yml

```diff
@@ -39,3 +42,5 @@ cache:
   ccache: true
   directories:
   - $HOME/install
+  - $HOME/mfem
+  - $HOME/occa
```

Comment:
Do we want these cached instead of relying on ccache?

Reply:
With PETSc, we are using a release and have logic to skip the install entirely (no configure, no make, no install; just use `$HOME/install/petsc-3.9.3`). We do not have that for MFEM or OCCA, both of which are built in-source using just `make`. We should delete these two lines. The builds will still be fast due to ccache.

Reply:
Updated. The run time is faster that I expected this time.